docs/csharp/programming-guide/strings/index.md

```diff
@@ -64,11 +64,23 @@ A string is an object of type <xref:System.String> whose value is text. Internal
 >  At compile time, verbatim strings are converted to ordinary strings with all the same escape sequences. Therefore, if you view a verbatim string in the debugger watch window, you will see the escape characters that were added by the compiler, not the verbatim version from your source code. For example, the verbatim string @"C:\files.txt" will appear in the watch window as "C:\\\files.txt".  
   
 ## Format Strings  
- A format string is a string whose contents can be determined dynamically at runtime. You create a format string by using the static <xref:System.String.Format%2A> method and embedding placeholders in braces that will be replaced by other values at runtime. The following example uses a format string to output the result of each iteration of a loop:  
+ A format string is a string whose contents are determined dynamically at runtime. Format strings are created by embedding *interpolated expressions* or placeholders inside of braces within a string. Everything inside the braces (`{...}`) will be resolved to a value and output as a formatted string at runtime. There are two methods to create format strings: string interpolation and composite formatting.
+
+### String Interpolation  
+Available in C# 6.0 and later, [*interpolated strings*](../../language-reference/tokens/interpolated.md) are identified by the `$` special character and include interpolated expressions in braces. If you are new to string interpolation, see the [String interpolation - C# interactive tutorial](../../tutorials/intro-to-csharp/interpolated-strings.md) for a quick overview.
```

Comment:
@hDoubleE sorry, I've missed it earlier: the second relative link is still broken. This time, because of not correct file extension: it must be `.yml` instead of `.md`.

If you wonder how I see what is broken, here is the "magic":
![image](https://user-images.githubusercontent.com/15279990/48603017-646b5000-e975-11e8-834f-ab66738b68e1.png)

Note the **OpenPublishing.Build** entry in the list of PR checks. If it has *Warnings* in status, select the [Details](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C16%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201811160105396336-9025%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ZZMX1Pf0BbAzXa0C7whg5UHE0UuVV2ADMArxY%2BCzpXc%3D&st=2018-11-16T01%3A05%3A02Z&se=2018-12-17T01%3A10%3A02Z&sp=r) link to see the report. From that report, you can learn what goes not perfect during the build of the files from your PR (ignore all other files, though).

Reply:
Added .yml and now it links to file in VS code. Hopefully put that to rest. I reviewed all the great links. The index.md file has a local snippet folder in the docs repo. Although tempted, I did not add mine to it. I added to docs/samples repo and linked the path to that location. Please verify code snippet path for correctness. Hopefully the `<snippet>` and .md link correctly. Thank you.

Reply:
Thanks for the tip. I'll use this after pushing to verify.

Reply:
> Although tempted, I did not add mine to it.

Great 👍